react-native-wp-text-input/ios/WPBackedTextInputView.swift

```diff
@@ -0,0 +1,38 @@
+import UIKit
+
+class WPBackedTextInputView: RCTBaseTextInputView {
+    lazy var _backedTextInputView: WPTextInptu = {
```

Comment:
Typo on the variable name... should be `WPTextInput`

Reply:
Absolutely!
Thanks for spotting that out :)

Reply:
Apparently I copied one name with the typo and pasted it all over the place 😛
Thanks @SergioEstevao for fixing the file names!